Email subscription API

vincentchiang1 opened the pull request:
<!--- Add a GIF that describes how you feel about this PR (or just a cool one) -->
![](https://media.giphy.com/media/TObbUke0z8Mo/giphy.gif?cid=ecf05e47avbbk1opjdj1k73y2gnj243fk9xue5q3ali6uxhv&rid=giphy.gif&ct=g)

## Description
Resolves #39, just the cloud function call, Kevo will hook it up to his button in his modal component.

## Other considerations
<!--- Workarounds, planned future changes, special notes, etc. -->

kevin-zou reviewed (commented):
I think the `package-lock.json` diff is coming from you being on an older version of node. Can you check what it is (`node -v` in terminal)? I think I'm on the latest version so whenever I install a package I just commit all the stuff you remove LOL. Seems to be supported by this: https://stackoverflow.com/questions/64813775/is-there-any-way-to-fix-package-lock-json-lockfileversion-so-npm-uses-a-specific

If it's too much of a hassle to figure out I think it's worth it to just cut the `package-lock.json` from this PR!

vincentchiang1 commented:
> I think the `package-lock.json` diff is coming from you being on an older version of node. Can you check what it is (`node -v` in terminal)? I think I'm on the latest version so whenever I install a package I just commit all the stuff you remove LOL. Seems to be supported by this: https://stackoverflow.com/questions/64813775/is-there-any-way-to-fix-package-lock-json-lockfileversion-so-npm-uses-a-specific
> 
> If it's too much of a hassle to figure out I think it's worth it to just cut the `package-lock.json` from this PR!

hmm I updated the other day to resolve the other issue and updated just now so I feel like i'm on the latest version? v14.17.5

kevin-zou commented:
> > I think the `package-lock.json` diff is coming from you being on an older version of node. Can you check what it is (`node -v` in terminal)? I think I'm on the latest version so whenever I install a package I just commit all the stuff you remove LOL. Seems to be supported by this: https://stackoverflow.com/questions/64813775/is-there-any-way-to-fix-package-lock-json-lockfileversion-so-npm-uses-a-specific
> > 
> > If it's too much of a hassle to figure out I think it's worth it to just cut the `package-lock.json` from this PR!
> 
> hmm I updated the other day to resolve the other issue and updated just now so I feel like i'm on the latest version? v14.17.5 @kevin-zou 

Ohhh I see, no worries then let's just send it 👍
